xcp_d/utils/doc.py

```diff
@@ -59,17 +59,17 @@
 """
 
 docdict[
-    "t1w_to_mni"
+    "t1w_to_template"
 ] = """
-t1w_to_mni : :obj:`str`
+t1w_to_template : :obj:`str`
     Path to the T1w-to-MNI transform file.
     May be "identity", for testing purposes.
 """
 
 docdict[
-    "mni_to_t1w"
+    "template_to_t1w"
```

Comment:
This is so cool. Not sure how I never knew about this functionality

Reply:
I stole it from nilearn